pkgs/os-specific/darwin/apple-source-releases/Libc/10.13.6.nix

```diff
@@ -0,0 +1,27 @@
+{ appleDerivation', stdenvNoCC, ed, unifdef, fetchzip }:
+
+appleDerivation' stdenvNoCC {
+  nativeBuildInputs = [ ed unifdef ];
+
+  patches = [
+    ./0001-Define-TARGET_OS_EMBEDDED-in-std-lib-io-if-not-defin.patch
+  ];
+
+  installPhase = let Libc_old = fetchzip {
+      url = "https://github.com/apple-oss-distributions/Libc/archive/refs/tags/Libc-997.90.3.tar.gz";
+      hash = "sha256-B18RNO+Rai5XE52TKdJV7eknosTZ+bRERkiU12d/kPU=";
+    };
+  in ''
+    export SRCROOT=$PWD
+    export DSTROOT=$out
+    export PUBLIC_HEADERS_FOLDER_PATH=include
+    export PRIVATE_HEADERS_FOLDER_PATH=include
+    bash xcodescripts/headers.sh
```

Comment:
```suggestion
    FEATURE_LEGACY_RUNE_APIS=1 FEATURE_LEGACY_UTMP_APIS=1 bash xcodescripts/headers.sh
```

Reply:
That will ensure `rune.h` and `utmp.h` are installed. These are included in the Apple SDKs but not installed with the libc sources by default. The lack of `utmp.h` broke inettools during the LLVM 16 update. It has a workaround, but I wanted to fix libc to install the missing header.